src/components/ChannelPreview/__tests__/ChannelPreview.test.js

```diff
@@ -14,15 +14,16 @@ import {
   getOrCreateChannelApi,
   getTestClientWithUser,
   useMockedApis,
+  dispatchMessageReadEvent,
+  dispatchMessageNewEvent,
+  dispatchMessageUpdatedEvent,
+  dispatchMessageDeletedEvent,
```

Comment:
can we alphabetize this object ordering?

Reply:
Done!!